### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Persdre**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -128,8 +128,8 @@ The following points explain the format of a command.
    e.g. `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG]` can be used as `c-add n/Betsy Crowe e/81234567 p/89007413 a/Blk 120 ABC Road t/Friday` 
     or as `c-add n/Betsy Crowe e/81234567 p/89007413 a/Blk 120 ABC Road`.
 
-* Items with `…​` after them can be used multiple times, including zero times.<br>
-   e.g. `[t/TAG] …​` can be used as ` ` (i.e. 0 times), `t/Friday`, `t/Friday t/PartTime`, etc.
+* Items with `...​` after them can be used multiple times, including zero times.<br>
+   e.g. `[t/TAG]...` can be used as `  ` (i.e. 0 times), `t/Friday`, `t/Friday t/PartTime`, etc.
```
Comment (yanbingtao) [inline docs/UserGuide.md]: there should be an empty space(" ") before "...".thanks

File: docs/UserGuide.md [hunk 2]
```diff
@@ -78,38 +78,38 @@ We have structured this User Guide in a way so that you can find what you need e
...
 
 The following table explains the technical terms used in this user guide.
 
 | Technical Term | What it means |
 | ---------------| --------------|
-| Prefix | The first word of a command. tCheck uses this command word to decide what kinds of actions it should perform |
-| Command Word | It is the input that follows behind a prefix. It is used to distinguish different kinds of parameters |
-| Parameter | Values given for specific operations |
+| Command Word | The first word of a command. tCheck uses this command word to decide what kinds of actions it should perform. |
+| Prefix | It is the character before every "/". It is used to distinguish different kinds of parameters. |
```
Comment (ureshiiYing) [inline docs/UserGuide.md]: instead of "character", perhaps use "character(s)" as some prefixes have more than one character

**Review by ureshiiYing — COMMENTED** [review]
Other than those parts commented, the rest are LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (Persdre) [reply docs/UserGuide.md]: thanks, I am changing it.

**Review by ureshiiYing — APPROVED** [review]
LGTM

**Review by yanbingtao — APPROVED** [review]

**Pull request merged**